src/assets/login/gift.svg

```diff
@@ -0,0 +1 @@
-.658-.828a.308.308 0 0 1 .079-.197c.059-.07.12-.091.153-.095 4.989-.27 8.262-3.858 9.707-6.066.095-.146.151-.315.161-.489.037-.614.159-1.332.39-1.852.242-.543.429-.545.504-.54.157.008.231.039.255.052.017.009.02.013.023.018h.001c.019.03.076.166.05.492-.053.65-.386 1.473-.64 1.956a1 1 0 0 0-.114.464c-.003 3.225 1.97 6.335 5.016 7.41a.432.432 0 0 1 .16.14c.043.061.047.098.048.1a.353.353 0 0 1-.067.2.236.236 0 0 1-.143.081l-.02.005c-3.744.924-6.111 4.266-7.852 7.085z" stroke="#2F354D" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path d="M174.537 48.497c-3.167-2.962-7.514-.024-9.755 2.617-.808.962-1.532 1.928-2.183 3.066-.744 1.301-.739 2.647-.744 4.16.029.926.808 1.897 1.909 1.626 1.19-.35 2.281-.454 3.397-.976 1.121-.606 2.242-1.213 3.436-1.646 1.366-.508 2.746-1.267 3.809-2.297 1.86-1.824 1.947-4.761.131-6.55z" fill="#E7EAF3"/><path d="M164.424 56.244a.84.84 0 0 0-.021-.07l-.001-.011c-.001-.027 0-.063.001-.111l.009-.167c.014-.227.034-.456.057-.697l.007-.075c.02-.213.041-.437.057-.657.034-.489.046-1.036-.06-1.577a3.23 3.23 0 0 0-.802-1.606c-.441-.477-1.037-.84-1.784-1.112a.383.383 0 0 1-.149-.133.306.306 0 0 1-.058-.187.35.35 0 0 1 .068-.204.236.236 0 0 1 .143-.082l.029-.007c2.291-.589 3.744-2.625 4.749-4.232a.53.53 0 0 0 .02-.034.234.234 0 0 1 .117-.093.29.29 0 0 1 .163-.02c.038.008.141.063.16.244.084 1.6.439 2.685 1.119 3.385.621.64 1.397.808 1.845.906l.113.025c.054.012.109.02.165.023.614.036 1.12.476 1.208.902a.314.314 0 0 1-.078.176.248.248 0 0 1-.15.094c-2.381.11-3.806 1.956-4.757 3.316-.146.21-.281.407-.407.592-.329.482-.602.883-.886 1.209-.404.461-.571.458-.591.457a.287.287 0 0 1-.168-.076.372.372 0 0 1-.118-.178z" stroke="#2F354D" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path d="M23.601 87.104c3.168-2.962 7.514-.024 9.756 2.618.807.96 1.532 1.927 2.183 3.065.744 1.301.739 2.647.743 4.16-.03.926-.808 1.897-1.909 1.626-1.19-.35-2.28-.453-3.397-.976-1.12-.606-2.242-1.212-3.436-1.646-1.366-.508-2.746-1.267-3.808-2.297-1.86-1.823-1.948-4.76-.132-6.55z" fill="#E7EAF3"/><path d="M34.012 87.505a1 1 0 0 1 .163 1.57c-1.129 1.08-2.04 2.281-2.601 3.765a1 1 0 0 1-1.597.395c-1.304-1.15-2.557-2.09-4.125-2.838a1.001 1.001 0 0 1-.114-1.742c1.075-.696 2.286-1.873 2.978-3.837a1 1 0 0 1 1.66-.364c1.13 1.163 2.41 2.285 3.636 3.05zM46.23 73.653c-1.794-.411-3.542-1.076-5.292-1.65a.844.844 0 0 1-.559-.978c.299-1.412.534-2.853.457-4.338v-.001l-3.752-1.712a.846.846 0 0 1-.458-1.004c.314-1.09.516-2.188.673-3.283l.001-.005c.066-.442.162-.879.23-1.32-1.69-.632-3.366-1.301-5.054-1.937a.843.843 0 0 1-.542-.806c.036-1.873.172-3.575.51-5.267M176.051 89.26c-1.786.45-3.645.651-5.465.936a.845.845 0 0 1-.942-.617c-.377-1.393-.824-2.783-1.569-4.071h0l-4.12.18a.847.847 0 0 1-.865-.686c-.217-1.113-.537-2.183-.895-3.23l-.002-.004c-.142-.424-.255-.857-.397-1.28-1.792.205-3.589.371-5.381.572a.843.843 0 0 1-.85-.471c-.82-1.684-1.474-3.262-1.943-4.922" stroke="#2F354D" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
How is it in night mode?

Reply:
@haritonasty 
![image](https://user-images.githubusercontent.com/14061779/75873426-ed372500-5e20-11ea-8d56-545313ccc6f1.png)

Reply:
Let's code svg with vars